### ISSUE TITLE
Setting position in gps plugin, vertical sensing, docimentation

### ISSUE THREAD

**Issue by mnissov** [ros-simulation/gazebo_ros_pkgs#1335]
Is there a documentation on the status of the gps plugin?

I've been trying to find some but aside from the demo world there doesn't seem to be much content, or even usage of the plugin. Understandably enough because it is lacking some relatively basic features. The first of which being the ability to set the initial position to something other than latitude/longitude (0,0).

Furthermore, as far as I can tell although there are optional arguments for vertical sensing in the sdf, it doesn't seem to be implemented.

In conclusion, is there some location with a current status on the gps plugin and what is desired for the future? And are my impressions regarding these features misplaced?

**Comment by mr337** [comment]
This isn't long term solution as I am trying to wrap my head around the new plugin as well but this gist helped [1]. Placing this in your world.sdf file will set the location.

```
<spherical_coordinates>
 <surface_model>EARTH_WGS84</surface_model>
 <latitude_deg>-37.644121</latitude_deg>
 <longitude_deg>176.183812</longitude_deg>
 <elevation>0</elevation> 
 <heading_deg>0</heading_deg> 
</spherical_coordinates>
```



1 - https://gist.github.com/mathewthomas916/b05dc0cc6518251b485b84d0e34a6063?permalink_comment_id=3974822#gistcomment-3974822
